schedule_lessons/templates/Schedulerpage.html

```diff
@@ -99,22 +101,58 @@ <h5 class="modal-title" id="exampleModalLabel">Edit Availability</h5>
                     </div>
 
                     <div class="modal-body">
-                        <div class="form-group">
+                        <div class="form-group text-center">
                             <h3>Day</h3>
-                            <label for="day" class="form-control-label"></label>
-                            <input type="text" class="form-control" id="day" placeholder="Monday">
-                        </div>
 
+                            <div class="btn-group">
+
+                                <button type="button" id="day" class="btn btn-info dropdown-toggle form-control dropdown-title" data-toggle="dropdown">
+                                  Select a day <span class="caret"></span>
+                                </button>
+                              <ul class="dropdown-menu" role="menu">
+                                <li><a class="list" href="#">Monday</a></li>
+                                <li><a class="list" href="#">Tuesday</a></li>
+                                <li><a class="list" href="#">Wednesday</a></li>
+                                <li><a class="list" href="#">Thursday</a></li>
+                                <li><a class="list" href="#">Friday</a></li>
+                                <li><a class="list" href="#">Saturday</a></li>
+                                <li><a class="list" href="#">Sunday</a></li>
+
+                              </ul>
+                            </div>
+                        </div>
                         <div class="form-group">
-                            <h3>Available Times</h3>
-                            <label for="availableTimes" class="form-control-label"></label>
-                            <input type="text" class="form-control" id="availableTimes" placeholder="5:00 PM - 8:00 PM">
+                          <h3>Timings</h3>
+                          <!--
+                            <form class="form-inline">
+
+                              <div class="form-group">
+
+                                <label for="defaultEntry1">Starting:</label>
+                                <input type="text" id="defaultEntry1" class="form-control" value="12:00">
+
+                              </div>
+                              <div class="form-group">
+                                <label for="defaultEntry"></label><br></br>
+                                <input type="text" id="defaultEntry" class="form-control" value="6:00">
+                              </div>
+                            </form>-->
+                            <form class="center form-inline ">
```

Comment:
what is that commented code for? if we dont need it anymore then you can remove it